BootloaderCommonPkg/Include/Library/CryptoLib.h

```diff
@@ -171,11 +171,9 @@ Sm3 (
 /**
   Verifies the RSA signature with PKCS1-v1_5 encoding scheme defined in RSA PKCS#1.
 
-  @param[in]  PubKey         Pointer to a pre-processed RSA key data.
-  @param[in]  Signature      Pointer to RSA PKCS1-v1_5 signature to be verified.
-  @param[in]  SignatureLen   Length of the signature in bytes.
-  @param[in]  SignatureType  Now only support signature type SIG_TYPE_RSA2048SHA256.
-  @param[in]  Hash           Pointer to octet message hash to be checked.
+  @param[in]  PubKeyHdr         Pointer to a PubKey data.
+  @param[in]  SignatureHdr      Pointer to signature data to be verified.
+  @param[in]  Hash              Pointer to octet message hash to be checked.
 
   @retval  RETURN_SUCCESS             Valid signature.
   @retval  RETURN_INVALID_PARAMETER   Key or signature format is incorrect.

```

Comment:
Can you add EFIAPI?

Reply:
Thanks. I will add. 
Also they need to be added public Hashfunctions as SHA256 and so. Would do as separate patch